AllJoyn/Platform/Bridge/BridgeCore/Bridge.vcxproj

```diff
@@ -288,8 +289,10 @@
       <Optimization>MaxSpeed</Optimization>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
-      <PreprocessorDefinitions>QCC_OS_GROUP_WINDOWS;NDEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>BRIDGE_WIN10;QCC_OS_GROUP_WINDOWS;_SCL_SECURE_NO_WARNINGS;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
arm - Centennial is desktop only. 
x86 - I can take a look.

---
In reply to: [75178260](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75178260) [](ancestors = 75178260,75177875,75174093)